msg/UnboundedSequencesNoStrings.msg

```diff
@@ -0,0 +1,32 @@
+# Unbounded sequences of different types
```

Comment:
@MiguelCompany if you have no use case for this message type, please don't add it.